justega247/favorite-things
# #167395729: Remove user authentication

#### What does this PR do?
- This PR is for the removal of the Authentication implementation to be in line with the current specifications
#### Description of Task to be completed?
- remove the user authentication in line with project specifications
- refactor existing code to reflect the new changes
#### How should this be manually tested?
- Start the application by going to the root directory and running **python manage.py runserver** 
- Navigate to any endpoint  and make a  request
- As long as the necessary details have been provided, the request should be successful without being prompted for authentication
#### What are the relevant pivotal tracker stories?
[#167395729](https://www.pivotaltracker.com/story/show/167395729)
